src/main/java/com/backbase/ct/bbfuel/configurator/AccessGroupsConfigurator.java

```diff
@@ -33,7 +34,7 @@
     private static final String ARRANGEMENTS = "ARRANGEMENTS";
 
     public JobProfile ingestAdminFunctionGroup(String externalServiceAgreementId) {
-        JobProfile adminProfile = new JobProfile(ADMIN_FUNCTION_GROUP_NAME, null, null, null, null);
+        JobProfile adminProfile = new JobProfile(ADMIN_FUNCTION_GROUP_NAME, null, null, null, null,Type.REGULAR.toString());
```

Comment:
- Line length is more than 120 characters
- Missing space before `Type.REGULAR.toString()`

Reply:
Done.